src/ui/server/test-mocks/mock-exporter-business.ts

```diff
@@ -0,0 +1,10 @@
+import { ExporterBusiness } from '../../types';
+
+const mockExporterBusiness = {
+  goodsOrServicesSupplied: 'ABC',
+  yearsExporting: '2,0',
+  employeesInternational: '1,000',
+  employeesUK: '4,00',
```

Comment:
Currently the test mocks for an application have actual data types that are saved in the Db rather than what's submitted via the UI so I think these new mocks should be numbers instead of strings with commas?